.github/workflows/contracts.yml

```diff
@@ -265,11 +265,13 @@ jobs:
           project_id: ${{ env.GOOGLE_PROJECT_ID }}
           service_account_key: ${{ secrets.KEEP_TEST_GCR_JSON_KEY }}
 
-      - name: Fetch external contracts artifacts
-        env:
-          CONTRACT_DATA_BUCKET_DIR: keep-ecdsa-celo
-          ETH_NETWORK_ID: ${{ env.NETWORK_ID }}
-        run: ./scripts/ci-provision-external-contracts.sh
+      - name: Get upstream packages' versions
+        uses: keep-network/upstream-builds-query@main
+        id: upstream-builds-query
+        with:
+          upstream-builds: ${{ github.event.inputs.upstream_builds }}
+          query: |
+            keep-ecdsa-solidity-version = github.com/keep-network/keep-ecdsa/solidity#version
```

Comment:
I think this should stay in the original form

Reply:
The old way will fail as we removed `scripts/ci-provision-external-contracts.sh` script. We need to revisit celo migrations and start publishing a package to npm.